DOCS.md

```diff
@@ -312,6 +312,11 @@ Example:<br />
 Journal example:<br />
   `{ j = { description = 'Journal', template = '\n*** %<%Y-%m-%d> %<%A>\n**** %U\n\n%?', target = '~/sync/org/journal.org' } }`
 
+Nested key example:<br />
+  `{ e =  'Event' }`
+  `{ r = { description = 'recurring', template = '** %?\n %T', target = '~/org/calendar.org', headline = 'recurring' } }`
```

Comment:
shouldn't keys be `er` and `eo` ?

Reply:
Absolutely, thanks for checking. Fixed it now